index.html

```diff
@@ -13,35 +13,189 @@
-                <li><a href="#"><img src="media/twitter.png" alt="twitter"></a></li>
-                <li><a href="#"><img src="media/medium.png" alt="medium"></a></li>          
+            <ul id="social-items" class="no-style-deco">
+                <li class="social-item"><a href="#"><img src="media/github.png" alt="github"></a></li>
+                <li class="social-item"><a href="#"><img src="media/Linkedin icon.png" alt="linkedIn"></a></li>
+                <li class="social-item"><a href="#"><img src="media/angel list.png" alt="angel list"></a></li>
+                <li class="social-item"><a href="#"><img src="media/twitter.png" alt="twitter"></a></li>
+                <li class="social-item"><a href="#"><img src="media/medium.png" alt="medium"></a></li>          
         </ul>
         </aside>
+        
         <div id="bio">
-        <h1>Hey There. <br> I'm Sylvester</h1>
-        <h2>I'm a Software Developer</h2>
-        <p>I can help you build a product , feature or website. Look through some of 
-            my work and experience! If you like what you see and have a project you need coded, <br>
+        <h1 class="orange-color">Hey There. <br> I'm Sylvester</h1>
+        <h2 class="white">I'm a Software Developer</h2>
+        <p class="white">I can help you build a product , feature or website. Look through some of 
+            my work and experience! If you like what you see and have a project you need coded,
             don't hesitate to contact me.</p>
         </div>
     </section>
+
+    <!-- Works section -->
+    <main>
+        
+        <h2 id="works" class="title">My Recent Works <hr></h2>      
+         
+                <img src="media/multipost.png" alt="Multipost image">
+        
+                <article class="grid-item" >
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="btn">
+                        <button type="button" class="grey-button">CSS</button>
+                        <button type="button" class="grey-button">HTML</button>
+                        <button type="button" class="grey-button">Bootstrap</button>
+                        <button type="button" class="grey-button">Ruby</button>
+                    </div>
+                    <button type="button" class="orange-button">See project</button>
+                </article>
+
+              <ul>
+   
+        <li class="grid-item white"> 
+            <div class="content">
+            <h2 class="title">Data Dashboard Healthcare</h2>
+            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
```

Comment:
- [ ] Here as well, 🙏 Please make sure to display all your projects in the **center of the page** as shown below:

#### figma design:
![f3](https://user-images.githubusercontent.com/105105262/227730178-4e7ec5d7-9215-420b-ae36-b52ea28026db.PNG)

#### your design:
![y3](https://user-images.githubusercontent.com/105105262/227730192-afb7b883-adf2-4be2-9ca1-21da004a7930.PNG)